plos_connector/plos_unittests.py

```diff
@@ -0,0 +1,96 @@
+import requests
```

Comment:
Would it be possible to test your custom exceptions with self.assertRaises method?

Reply:
added check for custom exceptions